umm-spec-lib/src/cmr/umm_spec/migration/version/variable.clj

```diff
@@ -3,13 +3,60 @@
   (:require
    [cmr.umm-spec.migration.version.interface :as interface]))
 
+(defn sample-11->12
```

Comment:
Why `11` and `12` rather than `1.1` and `1.2`? Maybe add a docstring or comment to explain the reasoning.

Reply:
Clojure doesn't let you put `.` in function names; that's parsed as a namespace/Class separator. Attempting to do so will result in `java.lang.ClassNotFoundException`.

Reply:
I find `11` and `12` confusing as well. 

I _think_ I prefer something like `sample-1-1->1-2`. What do you think about that format?
Or how about `sample-1dot1->1dot2`? Alright that last one is a joke.

Reply:
Done.